benchmarks/README.md

```diff
@@ -23,12 +23,55 @@ The OmniSafe Mujoco Velocity Benchmark assesses the efficacy of OmniSafe's SafeR
 - **[ICML 2017]** [Proximal Constrained Policy Optimization (PCPO)](https://proceedings.mlr.press/v70/achiam17a)
 - **[ICLR 2019]** [Reward Constrained Policy Optimization (RCPO)](https://openreview.net/forum?id=SkfrvsA9FX)
 
+
+**Saute RL**
+
+- **[ICML 2022]** [Sauté RL: Almost Surely Safe Reinforcement Learning Using State Augmentation (PPOSaute, TRPOSaute)](https://arxiv.org/abs/2202.06558)
+
+**Simmer**
+
+- **[NeurIPS 2022]** [Effects of Safety State Augmentation on Safe Exploration (PPOSimmerPID, TRPOSimmerPID)](https://arxiv.org/abs/2206.02675)
+
+**PID-Lagrangian**
+
+- **[ICML 2020]** [Responsive Safety in Reinforcement Learning by PID Lagrangian Methods(CPPOPID, TRPOPID)](https://arxiv.org/abs/2007.03964)
+
+**Early Terminated MDP**
+
+- **[Pre-Print 2021]** [Safe Exploration by Solving Early Terminated MDP(PPOEarlyTerminated, TRPOEarlyTerminated)](https://arxiv.org/pdf/2107.04200.pdf)
+
 > **More details can be refer to [On Policy Experiment](https://github.com/OmniSafeAI/omnisafe/tree/main/benchmarks/on-policy/README.md).**
 
 ## Off-Policy
+
 ### Supported Algorithms
-- [Deep Deterministic Policy Gradient (DDPG)](https://arxiv.org/pdf/1509.02971.pdf)
-- [Twin Delayed DDPG (TD3)](https://arxiv.org/pdf/1802.09477.pdf)
-- [Soft Actor-Critic (SAC)](https://arxiv.org/pdf/1812.05905.pdf)
+
+- **[ICLR 2016]** [Deep Deterministic Policy Gradient (DDPG)](https://arxiv.org/pdf/1509.02971.pdf)
+- **[ICML 2018]** [Twin Delayed DDPG (TD3)](https://arxiv.org/pdf/1802.09477.pdf)
+- **[ICML 2018]** [Soft Actor-Critic (SAC)](https://arxiv.org/pdf/1812.05905.pdf)
+- [The Lagrangian version of DDPG](https://cdn.openai.com/safexp-short.pdf)
+- [The Lagrangian version of TD3](https://cdn.openai.com/safexp-short.pdf)
```

Comment:
I didn't find these three algorithms in safexp-short.pdf which were provided via this hyper-link.

Reply:
This paper is safety-gym original paper. It's published code base [safety-starter-agents](https://github.com/openai/safety-starter-agents) implemented `SACLag` but does not report it in their paper.  I can not find the source of `DDPGLag `and `TD3Lag`. However, this paper introduced lagrangian methods and implemented `SACLag`, so I also use it as the source of `DDPGLag` and `TD3Lag`

Reply:
Got it, maybe we can describe this explicitly with the link.

Reply:
Copy that, I will.